docs(release): Final docs changes for the 1.19.0 release (#7092)
SeldonIO/seldon-core: docs-gb/wrappers/cpp.md

# Packaging a C++ Framework/Model for Seldon Core

{% hint style="warning" %}

> This wrapper is an example only and is not production ready.

Reviewer: ```suggestion
This wrapper is an example only and is not production-ready.
```

{% endhint %}

> In this guide we cover how you can wrap your CPP models using the Seldon CPP wrapper.

Reviewer: ```suggestion
In this guide we illustrate how you can wrap your CPP models using a CPP wrapper.
```
